src/build/eclipse/eclipse.importorder

```diff
@@ -0,0 +1,9 @@
+#Organize Import Order
+# Import this file in Window -> Preferences -> Java -> Code Style -> Organize Imports -> Import...
+0=
+# This project uses simple alphabetical import order
+# 1=java
+# 2=javax
+#
+# static imports
```

Comment:
this is all commented out - what does it acheive?

Reply:
Kept it to show the alternative settings, fine with removing.